platform/linux-generic/pktio/uapi_net_mdev.h

```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2018, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+#ifndef _UAPI__LINUX_NET_MDEV_H
+#define _UAPI__LINUX_NET_MDEV_H
+
+#include <linux/types.h>
+#include <linux/ioctl.h>
+
+enum net_mdev_types {
+	VFIO_NET_MDEV_SHADOW,
+	VFIO_NET_DESCRIPTORS,
+	VFIO_NET_MMIO,
+};
+
+enum net_mdev_subtypes {
+	VFIO_NET_MDEV_STATS,
+	VFIO_NET_MDEV_RX,
+	VFIO_NET_MDEV_TX,
+	VFIO_NET_MDEV_BARS,
+};
```

Comment:
New separate PR for common code (patch #1 in this PR) is created: https://github.com/Linaro/odp/pull/384
This PR and other driver-specific ones will be rebased as soon as #384 gets merged.